config/environments/development.rb

```diff
@@ -3,37 +3,27 @@
 require 'active_support/core_ext/integer/time'
 
 Rails.application.configure do
-  config.after_initialize do
-    Bullet.enable        = true
```

Comment:
why remove Bullet?

Reply:
you are right, it got overwritten when I ran the `app:update` task. Ill add it back